### PR TITLE
issue: 2583763 Extend ci with ubuntu 22.04 

### PR DESCRIPTION
**Pull request by AoD314**
## Description
Add ubuntu 22.04 to CI

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

**Comment by AoD314** [comment]
bot:retest

**Comment by alexbriskin** [comment]
@AoD314 , please explain how this PR will capture the XML/HTML artifacts.

File: contrib/jenkins_tests/globals.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ commit_dir=${WORKSPACE}/${prefix}/commit
 prj_lib=libvma.so
 prj_service=vmad
 
+# make_opt="-j8"
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/globals.sh:32]: please remove

File: contrib/jenkins_tests/cov.sh [hunk 2]
```diff
@@ -25,6 +25,7 @@ sleep 1
 eval "cov-configure --config ${cov_dir}/coverity_config.xml --gcc >> ${cov_dir}/cov.log 2>&1"
 sleep 1
 eval "cov-build --config ${cov_dir}/coverity_config.xml --dir ${cov_build} make -j >> ${cov_dir}/cov.log 2>&1"
+# eval "cov-build --config ${cov_dir}/coverity_config.xml --dir ${cov_build} make -j8 >> ${cov_dir}/cov.log 2>&1"
```
[the remark this answers is not in the document]
Comment (AoD314) [reply contrib/jenkins_tests/cov.sh]: fixed

File: contrib/jenkins_tests/globals.sh (hunk 1 above)
Comment (AoD314) [reply contrib/jenkins_tests/globals.sh]: fixed

File: contrib/jenkins_tests/rpm.sh [hunk 3]
```diff
@@ -27,6 +27,7 @@ else
     echo "Build rpms"
     opt_rpm=1
     rpmspec=${build_dir}/0/contrib/scripts/libvma.spec
+    # --define='%_smp_mflags -j8' \
```
[the remark this answers is not in the document]
Comment (AoD314) [reply contrib/jenkins_tests/rpm.sh]: fixed

**Review by lennybe — APPROVED** [review]

**Review by lennybe — APPROVED** [review]

File: contrib/jenkins_tests/cov.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (lennybe) [reply contrib/jenkins_tests/cov.sh]: @igor-ivanov , please review and approve

**Comment by igor-ivanov** [comment]
bot:retest

**Comment by AoD314** [comment]
bot:retest

**Review by vasily-v-ryabov — APPROVED** [review]

**Pull request merged**